modules/mkr/components/Withdraw.tsx

```diff
@@ -115,6 +115,8 @@ const ModalContent = ({ address, voteProxy, close, ...props }) => {
 
             const txId = await track(freeTxCreator, 'Withdrawing MKR', {
               mined: txId => {
+                // Mutate locked amount
+                mutateLocked();
```

Comment:
This is to mutate and refresh the total locked MKR in the contract, was not working before